### PR TITLE
Automated pipelines update

### PR DESCRIPTION
**Pull request by pierDipi**
This will keep pipelines up to date

We need to inject parameters and for that we can use Kustomizations

### CODE REVIEW

File: .github/workflows/release-discover-branches.yaml [hunk 1]
```diff
@@ -30,6 +29,13 @@ jobs:
         with:
           go-version-file: ./src/github.com/openshift-knative/hack/go.mod
 
+      - uses: tektoncd/actions/setup-tektoncd-cli@main
+        with:
+          version: latest
+
+      - name: Install Kustomize
+        run: GO111MODULE=on go install sigs.k8s.io/kustomize/kustomize/v5@latest
```
Comment (creydr) [inline .github/workflows/release-discover-branches.yaml:37]: nit: kustomize is already in ubuntu-latest runner image installed (https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#tools)

Comment (pierDipi) [reply .github/workflows/release-discover-branches.yaml:37]: I think having a specific version helps?

**Review by creydr — COMMENTED** [review]
nice. Only one nit. But feel free to unhold it otherwise

/lgtm
/hold

**Comment by pierDipi** [comment]
/unhold

I think it's fine for now

**Pull request merged**